microsoft/testsuites/xfstests/xfstesting.py

```diff
@@ -548,6 +568,9 @@ def _execute_xfstests(
             cwd=xfstests.get_xfstests_path(),
```

Comment:
I just found this is the actual xfstests command. It should be wrapped in the xfstests, not compose a command in test case level.

Reply:
update